### PR TITLE
Add next meeting

### PR DESCRIPTION
**Pull request by ml-evs**
Not entirely sure when the next meeting was confirmed for, but there was a suggestion that 28th Sept was confirmed. Could @gmrigna or @rartino perhaps confirm? (I assume we will have meetings about the paper sooner than that...)

### CODE REVIEW

**Comment by rartino** [comment]
I have a vague memory that we said to decide the next meeting at the end of the workshop. I had to miss the final day. (Sept 28 would work for me, though.)

File: _includes/next_meeting.html [hunk 1]
```diff
@@ -1 +1 @@
-[Friday May 6th 2022 at 15:00 UTC/17:00 CET](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220506T15){:target="_blank"}
+[Wednesday September 28th 2022 at 15:00 UTC/17:00 CET](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220928T15){:target="_blank"}
```
Comment (ml-evs) [inline _includes/next_meeting.html:1]:
```suggestion
[Wednesday September 28th 2022 at 14:00 UTC/16:00 CET](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220928T14){:target="_blank"}
```

**Review by merkys — APPROVED** [review]
Yes, this was the date proposed on the last day of the Lausanne meeting.

**Comment by ml-evs** [comment]
Thanks both! (and thanks @JPBergsma for giving me the date in the first place).

**Pull request merged**